### DISCUSSION TITLE
TypeError: Cannot set property 'nette-submittedBy' of undefined

### DISCUSSION THREAD

**Discussion by EncryptSL** [jiripudil/Naja#182] [Support] [answered]
Hi @jiripudil, i have linked naja.min.js from latest build in project but when i click on element with ajax in class, Script call this error. 

FormsHandler.ts:39 Uncaught TypeError: Cannot set property 'nette-submittedBy' of undefined
    at w.processForm (FormsHandler.ts:39)
    at EventTarget.clickElement (UIHandler.ts:82)
    at EventTarget.handleUI (UIHandler.ts:75)

![image](https://user-images.githubusercontent.com/9441083/102852578-dbaa2280-441e-11eb-9a76-905b2741552f.png)

**Comment by jiripudil** [comment, marked as answer]
Hi, that looks like a bug and I have a pretty good suspicion about the cause. Thanks for the report, let's follow up in #183

**Reply by EncryptSL** [reply]
Thanks for fix ;)
